pages/en/lb4/Preparing-the-API-for-consumption.md

```diff
@@ -14,11 +14,23 @@ This article continues off from [Implmenting features](./Implementing-features.
 
 ## Preparing your API for consumption
 
-{% include content/tbd.html %}
+### Interacting with you API
 
-- Documentation
-- Public endpoint for the swagger spec (for discovery)
-- more?
+When your application is run, its documentation is generated into `swagger.json` file and can be accessed from your server through `/swagger.json`. You can also interact with the API endpoints defined in `swagger.json` file through Swagger UI (`/swagger-ui`), which allows users to directly interact with the documented API endpoints of the application.
+
+We'll use the repo [loopback4-example-getting-started](https://github.com/strongloop/loopback4-example-getting-started) to demonstrate how Swagger UI can be used to test your endpoints.
+
```

Comment:
nitpick feel free to ignore: I think adding something to hint users run the app here would be better, like 
- `git clone https://github.com/strongloop/loopback4-example-getting-started`
- `npm i`
- `npm start`

Or just "run the app"